server/user.js

```diff
@@ -0,0 +1,80 @@
+const { TimeLogger } = require("../src/util");
+const { R } = require("redbean-node");
+const { UptimeKumaServer } = require("./uptime-kuma-server");
+const server = UptimeKumaServer.getInstance();
+const io = server.io;
+
+/**
+ * Send list of users to client
+ * @param {Socket} socket Socket.io socket instance
+ * @returns {Promise<Bean[]>}
+ */
+async function sendUserList(socket) {
+    const timeLogger = new TimeLogger();
+
+    const userList = await R.getAll("SELECT id, username, active FROM user");
+
+    io.to(socket.userID).emit("userList", userList);
+
+    timeLogger.print("Send User List");
+
+    return userList;
+}
+
+/**
+ * Send list of users to client
+ * @param {Socket} socket Socket.io socket instance
+ * @returns {Promise<Bean[]>}
+ */
```

Comment:
Could you update this JSDoc comment to match the function please

```suggestion
/**
 * Fetch specified user
 * @param {number} userID ID of user to retrieve
 * @returns {Promise<Bean[]>} User
 */
```

Reply:
Oups! 🙈

Reply:
done